ui/tests/unit/services/analytics-test.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test, skip } from 'qunit';
+
+import sinon from 'sinon';
+
+import { setupTest } from 'vault/tests/helpers';
+
+class ProviderStub {
+  name = 'testing';
+
+  start = sinon.stub();
+  identify = sinon.stub();
+  trackPageView = sinon.stub();
+}
+
+module('Unit | Service | analytics', function (hooks) {
+  setupTest(hooks);
+
+  hooks.beforeEach(function () {
+    this.service = this.owner.lookup('service:analytics');
+  });
+
+  hooks.afterEach(function () {
+    sinon.reset();
+  });
+
+  module('#start', function () {
+    skip('#start does nothing when no provider is given', function (assert) {
```

Comment:
is there a plan to un-skip these at some point?

Reply:
I wanted to test the start mechanism a differently than I got time for. I'll remove these for now and come back to them after code freeze